### ISSUE TITLE
Enable Item-level versioning in DSpace 6

### ISSUE THREAD

**Issue by alawvt** [VTUL/vtechworks#626]
Enable Item-level versioning in DSpace 6

[DSpace 6 Wiki: Item Level Versioning](https://wiki.duraspace.org/display/DSDOC6x/Item+Level+Versioning)

Note in https://github.com/VTUL/vtechworks/blob/vt_6_x_dev/dspace/config/xmlui.xconf
```
<!-- =====================
             Item Level Versioning
             ===================== -->
        <!--
             To enable Item Level Versioning features, uncomment this aspect.
             This is currently disabled by default because of one known issue:
             DS-1382. Please, review them to see whether they apply
             to you before enabling versioning.
        -->
<!--<aspect name="Versioning Aspect" path="resource://aspects/Versioning/" />-->
```
See #615 regarding the  desired Handle numbering system for versions

**Comment by alawvt** [comment]
Resolved with #627 and deployed manually to vtechworks-dev.

**Comment by keithgee** [comment]
Thank you for correcting this configuration, @alawvt.
